### PR TITLE
551_show_tag_title_on_cards_overview

### PR DESCRIPTION
**Pull request by philippe-levan**
Signed-off-by: Philippe Le Van <philippe.levan@kibatic.com>

* Fixes issue: #551
* Target version: master 

### Summary

cards labels visible under the title

image

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
N/A (only UI)
- [ ] Documentation (manuals or wiki) has been updated or is not required
N/A (only UI)

### CODE REVIEW

File: css/style.scss [hunk 1]
```diff
@@ -509,37 +509,28 @@ input.input-inline {
...
-		left: 10px;
+		position: relative;
+		margin-left: 5px;
 
 		li {
-			padding: 0;
-			width: 15px;
-			height: 20px;
+			padding: 0 4px;
+			margin: 0 2px 1px 0;
```
Comment (juliushaertl) [inline css/style.scss]: Lets probably use a 2px bottom margin as well, so they are nicly aligned when using multiple lines.

**Comment by juliushaertl** [comment]
Look really good. Just one small comment. :wink: 

cc @nextcloud/deck

File: css/style.scss (hunk 1 above)
Comment (philippe-levan) [reply css/style.scss]: done :-)

**Comment by juliushaertl** [comment]
Thanks a lot, this is a big improvement :+1:

**Comment by philippe-levan** [comment]
Thanks for the review and the merge !

**Pull request merged**